src/components/Dialogs/utils.tsx

```diff
@@ -29,6 +30,15 @@ const metricToLearnLink: { [key in Metric]: string } = {
     '/covid-community-level-metrics#weekly-new-cases',
 };
 
+const howItsGradedMap: { [key: number]: string } = {
+  [Metric.ADMISSIONS_PER_100K]:
+    'Levels for weekly new admissions are based on the CDC’s Community Level framework. Less than 10 weekly new admissions is considered Low, between 10 and 19.9 is considered Medium, and greater than or equal to 20 is considered High. No color grading is applied prior to the April 18, 2022 change to a community risk level framework, which better reflects the more recent decreased risk of severe illness and death from COVID due to vaccines, therapeutics, and past COVID infections, and other developments.',
+  [Metric.RATIO_BEDS_WITH_COVID]:
+    'Levels for percentage of patients with COVID are generally based on the CDC’s Community Level framework. Less than 10% of all new beds is considered Low, between 10 and 14.9% is considered Medium, and greater than or equal to 15 is considered High. No color grading is applied prior to the April 18, 2022 change to a community risk level framework, which better reflects the more recent decreased risk of severe illness and death from COVID due to vaccines, therapeutics, and past COVID infections, and other developments.',
```

Comment:
Any idea why this says "are generally based on" and the other two say "are based on" ?

Reply:
Seems like "equal to 15" should be "equal to 15%"

Reply:
not sure why it says "generally" in this case, I'm going to leave that, but update 15 → 15% 👍